src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.17.1"
```

Comment:
@gaya3-zipstack why are we we treating this as a breaking change? Or is it because we are introducing a new feature of `context_window_size`? If this can be a patch fix, it'll be easier to update downstream